expanderr.go

```diff
@@ -349,7 +349,7 @@ func (e *expansion) typeCheck(pkgname string, files []*ast.File) error {
 	conf := types.Config{
 		Importer: defaultImporter(),
 		Error: func(err error) {
-			log.Printf("ignoring type-checking error: %v", err)
+			// log.Printf("ignoring type-checking error: %v", err)
```

Comment:
Why are you commenting out this line?

If errors should be swallowed here, that deserves an explanatory comment.

Reply:
As far as I can tell this is just a debug log. It is printed out but then everything keeps going and nothing is done with the fact that there was an error here.

Reply:
When I wrote that line, I wasn’t sure whether ignoring type-checking errors is the correct thing to do. I’m still not sure, so I figure we should tell users what we’re doing.

Reply:
At first I was starting to try and accumulate them as warnings and return those at the end before just discarding them as I did here. I'll see if I can get that working later.